browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
```

Comment:
Curious where this comes up—are there one or two recurring variables in particular in the app that make this rule problematic? Generally speaking this is a pretty good rule to have enabled, there are a few super confusing cases it prevents.

If it's just one or two variables you can override them in the project specifically using `capIsNewExceptions` and `newIsCapExceptions` (http://eslint.org/docs/rules/new-cap)